src/postamble.js

```diff
@@ -244,7 +244,9 @@ function stackCheckInit() {
   _emscripten_stack_init();
 #endif
   writeStackCookie();
+  stackCheckInit.done = true;
 }
+stackCheckInit.done = false;
```

Comment:
The `stackCheckInit.done` part may be separable out? I ask because if we find it causes any issues, it might be nicer for bisection.

Reply:
Done.